deploy/05_vote.ts

```diff
@@ -0,0 +1,33 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { DeployFunction } from "@celo/staked-celo-hardhat-deploy/types";
+import { catchNotOwnerForProxy, executeAndWait } from "../lib/deploy-utils";
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deploy } = hre.deployments;
+  const { deployer } = await hre.getNamedAccounts();
+
+  const isVoteAlreadyDeployed = await hre.deployments.getOrNull("Vote");
+
+  const managerAddress = (await hre.deployments.get("Manager")).address;
+  const deployment = await catchNotOwnerForProxy(
+    deploy("Vote", {
+      from: deployer,
+      log: true,
+      proxy: {
+        proxyArgs: ["{implementation}", "{data}"],
+        proxyContract: "ERC1967Proxy",
+        execute: {
+          init: {
+            methodName: "initialize",
+            args: [hre.ethers.constants.AddressZero, deployer, managerAddress],
+          },
+        },
+      },
+    })
+  );
+};
+
+func.id = "deploy_vote";
+func.tags = ["Vote", "core", "proxy", "voteDeploy"];
```

Comment:
What is the diference between `vote` and `voteDeploy`? Could we only use `vote`?

Reply:
I would prefer to leave it as it is since we use this approach consistently:

- Vote: Only vote contract
- voteDeploy: All deploy scripts that are necessary to deploy vote contract + setup